### ISSUE TITLE
Iterator was called on Null.

### ISSUE THREAD

**Issue by rishabhdeepsingh** [rodydavis/data_tables#8]
When I try to build this widget simply by using the below code. I am getting the following error. any help would be appreciated.
```dart
    return NativeDataTable(
      columns: [
        DataColumn(label: Text('Name')),
        DataColumn(label: Text('Address')),
      ],
      rows: [
        DataRow(cells: [
          DataCell(Text('this is a name')),
          DataCell(Text('This is an address')),
        ])
      ],
    );
```

```
The getter 'iterator' was called on null.
Receiver: null
Tried calling: iterator
```

**Comment by rodydavis** [comment]
Working on a refactor again the moment, but make sure they data is not null. Check out the example too

**Comment by rishabhdeepsingh** [comment]
I know the data is not null. still getting this error.

**Comment by bas24** [comment]
Any progress on this ASAP?

**Comment by rodydavis** [comment]
I fixed this in version `1.2.0`
